docs/src/pages/demos/dialogs/ConfirmationDialog.js

```diff
@@ -141,14 +149,7 @@ function ConfirmationDialog() {
         <ListItem button divider disabled>
           <ListItemText primary="Default notification ringtone" secondary="Tethys" />
         </ListItem>
-        <ConfirmationDialogRaw
-          classes={{
```

Comment:
Why not using the classes API here?

Reply:
Got you covered. I think I understand the motivation for this. Working on a different TS approach.